econml/inference.py

```diff
@@ -351,14 +355,17 @@ def const_marginal_effect_interval(self, X, *, alpha=0.1):
     def const_marginal_effect_inference(self, X):
         if (X is not None) and (self.featurizer is not None):
             X = self.featurizer.transform(X)
-        pred = np.array([mdl.predict(X) for mdl in self.fitted_models_final])
-        if not hasattr(self.fitted_models_final[0], 'prediction_stderr'):
-            raise AttributeError("Final model doesn't support prediction standard eror, "
-                                 "please call const_marginal_effect_interval to get confidence interval.")
-        pred_stderr = np.array([mdl.prediction_stderr(X) for mdl in self.fitted_models_final])
-        return NormalInferenceResults(d_t=self.d_t, d_y=self.d_y, pred=np.moveaxis(pred, 0, -1),
-                                      # send treatment to the end, pull bounds to the front
-                                      pred_stderr=np.moveaxis(pred_stderr, 0, -1), inf_type='effect',
+        pred = np.moveaxis(np.array([mdl.predict(X) for mdl in self.fitted_models_final]), 0, -1)
+        if hasattr(self.fitted_models_final[0], 'prediction_stderr'):
```

Comment:
I guess it's safe to assume that if the first model has this attribute, they all do?

Reply:
yeah this is used in the case where a single same model is trained for separate targets